Formula/clickhouse.rb

```diff
@@ -0,0 +1,74 @@
+class Clickhouse < Formula
+  desc "Free analytics DBMS for big data with SQL interface"
+  homepage "https://clickhouse.tech"
+  url "https://github.com/ClickHouse/ClickHouse/releases/download/v21.6.3.14-stable/ClickHouse_sources_with_submodules.tar.gz"
+  version "21.6"
+  sha256 "696bbcdef4796bfdf1becdb78abb5e97c0220acc9863f5a821193dd297ce428b"
+  license "Apache-2.0"
+  head "https://github.com/ClickHouse/ClickHouse.git"
```

Comment:
```suggestion
  head "https://github.com/ClickHouse/ClickHouse.git"

```
Let's make the formatting here consistent with other formulae.

Reply:
Got it. I'll add it after I'll get current build results.